app/assets/stylesheets/pages/involvement.scss

```diff
@@ -0,0 +1,89 @@
+/* Involvement Styles */
+
+.involvement_page {
+
+  .container {
+    display: flex;
+    flex-direction: column;
+    padding-left: 5cm;
+    padding-right: 5cm;
+    margin-top: 2cm;
```

Comment:
@sonia-y why not just be explicit and use 
```
padding-left: 5px;
padding-right: 5px;
```? 

if you have just `padding-left: 0px 5px` it's not as explicit to the reader